src/index.jsx

```diff
@@ -52,7 +52,7 @@ subscribe(APP_READY, () => {
           <Route path={ROUTES.Detail.EDIT} element={<LibraryEditPage />} />
           <Route path={ROUTES.Detail.ACCESS} element={<LibraryAccessPage />} />
           <Route path={ROUTES.Detail.IMPORT} element={<CourseImportPage />} />
-          <Route path={`${ROUTES.Block.HOME}/*`} element={<LibraryBlockPage />} />
```

Comment:
@kdmccormick: I lost track of this when https://github.com/openedx/frontend-platform/pull/574 seemed to make things work on tutor. But I'm wondering if this could be the cause of the edit page still being broken on Tutor dev?